lib/Data/Time/Calendar/HasDays.hs

```diff
@@ -0,0 +1,8 @@
+{-# LANGUAGE Safe #-}
+
+module Data.Time.Calendar.HasDays (
+    HasDays (..),
+) where
+
+import Data.Time.Calendar.HasDays.Class
```

Comment:
I tried to replicate the layout of other typeclasses here, such as `Data.Time.Format`.